types.d.ts

```diff
@@ -5,12 +5,12 @@
  */
 export type License = SPDXLicense | CustomLicense;
 
-type SPDXLicense = {
+export type SPDXLicense = {
   type: string;
-  url?: string;
+  url: string;
```

Comment:
I think that we should just remove this `url` field as SPDX licenses doesn't need urls, right?

Reply:
Nope. The URL does exist when you access it in this way:

```js
import * as icons from 'simple-icons';

console.log(icons['siApache']).license;
//=> {type: 'Apache-2.0', url: 'https://spdx.org/licenses/Apache-2.0'}
```

See here: https://github.com/simple-icons/simple-icons/blob/develop/scripts/build/package.js#L49